app/src/main/java/com/ncnf/repositories/OrganizationRepository.java

```diff
@@ -37,46 +36,19 @@ public OrganizationRepository(FirebaseDatabase db)  {
     }
 
     /**
-     * Loads the Organization from Database
-     * @param uuid the unique identifier of the Organization
-     * @return A CompletableFuture wrapping the loaded Organization
-     */
-    public CompletableFuture<Organization> loadOrganization(String uuid){
-        return this.db.getDocument(ORGANIZATIONS_COLLECTION_KEY + uuid, Organization.class);
-    }
-
-    /**
-     * Stores the Organization to Database
-     * @param organization the Organization object to store
-     * @return A CompletableFuture wrapping a boolean indicating that the request was successful or not
-     */
-    public CompletableFuture<Boolean> storeOrganization(Organization organization){
-        return this.db.setDocument(ORGANIZATIONS_COLLECTION_KEY + organization.getUuid(), organization);
-    }
-
-    /**
-     * Loads multiple Organization objects from Database
-     * @param uuidList the list of Organization unique identifiers to load
-     * @return A CompletableFuture wrapping a list containing the loaded Organization objects
-     */
-    public CompletableFuture<List<Organization>> loadMultipleOrganizations(List<String> uuidList){
-        return this.db.whereIn(ORGANIZATIONS_COLLECTION_KEY, UUID_KEY, uuidList, Organization.class);
-    }
-
-    /**
-     * Get the organization a user belongs to
-     * @param uuid the user identifier
-     * @return a future organization
```

Comment:
why did you remove all these methods ? Were we not using them ?

Reply:
I think when doing the automatic refactoring intellIJ suggested to erase it since it wasn't used so we might put it back later yes.